pandas/tests/indexing/multiindex/test_getitem.py

```diff
@@ -1,11 +1,11 @@
 import numpy as np
 import pytest
 
-from pandas.compat import StringIO, lrange, range, u, zip
+from pandas.compat import range, u, zip
 
 import pandas as pd
 from pandas import DataFrame, Index, MultiIndex, Series
-import pandas.core.common as com
```

Comment:
our linting checks are configured to enforce that anything used from `pandas.core.common` is imported in this manner